lib/public/Search/IProvider.php

```diff
@@ -68,15 +68,16 @@ public function getName(): string;
 	/**
 	 * Get the search provider order
 	 * The lower the int, the higher it will be sorted (0 will be before 10)
+	 * If null, the search provider will be hidden
```

Comment:
```suggestion
	 * If null, the search provider will be hidden in the UI and the API not called
```

Otherwise could be read as I'm still being called/invoked